reactor-core/src/main/java/reactor/core/publisher/ContextPropagation.java

```diff
@@ -380,90 +414,5 @@ public Iterator<T> iterator() {
 			this.body = body;
 			this.contextSnapshot = contextSnapshot;
 		}
-
-	}
-
-	static final class ContextCaptureNoPredicate implements Function<Context, Context> {
```

Comment:
As part of this change I moved this class above the Queue wrappers and closer to the context capture operator logic.